core-bundle/src/Resources/contao/classes/BackendUser.php

```diff
@@ -259,15 +250,15 @@ public function hasAccess($field, $array)
 			$field = array($field);
 		}
 
-		if (\is_array($this->$array) && array_intersect($field, $this->$array))
+		if (\is_array($this->arrData[$array]) && array_intersect($field, $this->arrData[$array]))
```

Comment:
All these usages need a null check to avoid undefinded array key errors I think, right?

```suggestion
		if (\is_array($this->arrData[$array] ?? null) && array_intersect($field, $this->arrData[$array]))
```